corehq/toggles.py

```diff
@@ -1660,6 +1660,15 @@ def _commtrackify(domain_name, toggle_is_enabled):
     relevant_environments={'icds', 'icds-new', 'india'},
 )
 
+
+ICDS_DISHA_API_V2 = StaticToggle(
+    'icds_disha_access',
+    'ICDS: Access New DISHA API V2',
```

Comment:
Should update this to say something like `Dashboard Indicator API for NIC`

Reply:
cool sounds good.